modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/runtime/source/PollingSourceWrapperTestCase.java

```diff
@@ -138,72 +133,82 @@ public void idempotencyStoreGetsCreatedOnStart() throws MuleException {
 
   @Test
   public void loggingOnAcceptedItem() throws MuleException, Exception {
+    logger.resetLogs();
+    logger.setLevel(DEBUG);
     stubPollItem(Collections.singletonList(null), Collections.singletonList(null));
-    logger = createMockLogger(debugMessages, DEBUG);
     startSourcePollWithMockedLogger();
-    verifyLogMessage(debugMessages, PollingSourceWrapper.ACCEPTED_ITEM_MESSAGE, "");
+    verifyLogMessage(logger.getMessages(), PollingSourceWrapper.ACCEPTED_ITEM_MESSAGE, "");
+    logger.resetLevel();
```

Comment:
put this in an after of the test so that it will always be invoked even if an exception happens.